### DISCUSSION TITLE
Сreate an Extension for Raycast

### DISCUSSION THREAD

**Discussion by in0vik** [MonitorControl/MonitorControl#1621] [General]
### Before opening the issue, have you...?

- [x] Searched for existing issues

### Is your feature request related to a problem? Please describe

I frequently use MonitorControl to adjust the brightness and volume of external monitors connected to my Mac. However, switching between different apps or using keyboard shortcuts can sometimes be cumbersome, especially when multitasking. There’s no seamless way to integrate MonitorControl’s functionality directly into Raycast, which would allow for quick adjustments without leaving the Raycast command interface. This creates a break in workflow and reduces overall productivity.

### Describe the solution you'd like

I would like to see an extension for Raycast that integrates MonitorControl. This extension should allow users to adjust external monitor brightness, contrast, and volume directly from within Raycast by typing specific commands (e.g., “Adjust Brightness”). It should offer real-time feedback and the ability to switch between multiple monitors if more than one is connected. Ideally, this extension would also provide customization options for assigning default commands or shortcuts to commonly used settings.

### Describe alternatives you've considered

I’ve considered relying on MonitorControl’s built-in macOS menu bar interface and using native macOS keyboard shortcuts, but these methods are less efficient in comparison to the streamlined experience that Raycast could provide. I’ve also explored creating custom Apple Shortcuts or Automator scripts, but these options lack the real-time feedback and ease of use that a Raycast extension could deliver.

### Anything else?

https://www.raycast.com/developers

**Comment by waydabber** [comment]
I recommend using BetterDisplay's CLI and integration features.

https://github.com/waydabber/BetterDisplay/wiki/Integration-features,-CLI

**Comment by Erik5000** [comment]
I would LOVE to get an Raycast extension for monitor control! Loving both apps!
